vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -426,6 +380,12 @@ ${(commandOutputJson as CommandExecutorResult).stderr}`));
 with options ${JSON.stringify(options)}.`));
             const commandResult = proc.spawnSync(command.commandRoot, command.commandParts, options);
 
+            if(os.platform() === 'win32')
+            {
+                proc.spawn('taskkill', ['/pid', commandResult.pid.toString(), '/f', '/t']);
```

Comment:
this is here to try to kill a timed out installer. will not work without privelleges.

Reply:
Does `spawn` care about the result of the operation?

Reply:
It does not.